src/Microsoft.CodeAnalysis.Analyzers/Test/MetaAnalyzers/InvalidReportDiagnosticRuleTests.cs

```diff
@@ -53,13 +53,23 @@ private static void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
         Diagnostic diag = Diagnostic.Create(descriptor2, Location.None), diag2 = Diagnostic.Create(descriptor2, Location.None); 
         context.ReportDiagnostic(diag);
     }
+
+    private static void AnalyzeOperation(OperationAnalysisContext context)
+    {
+        context.ReportDiagnostic(Diagnostic.Create(descriptor2, Location.None));
+
+        var diag = Diagnostic.Create(descriptor2, Location.None);
+        context.ReportDiagnostic(diag);
+    }
```

Comment:
Yes, because this is a syntax analyzer and there is a difference in descriptor variable passed in nested invocation versus flowing via a local.